### DISCUSSION TITLE
IOS WEbdriveio appium  not able to interact with google maps popup

### DISCUSSION THREAD

**Discussion by mahlakoks** [webdriverio/webdriverio#13617] [Technical Discussion]
I am not able to interact with the google maps pop up, I am not able to inspect the map so I tried below code but it's not working. Does anyone have a solution.

  const allowOnceButton = await $('~Allow Once'); 
   const allowWhileUsingButton = await $('~Allow While Using App'); 
   const dontAllowButton = await $('~Don\'t Allow'); 

   const maxAttempts = 5;
   let attempts = 0;

   while (attempts < maxAttempts) {
       // Wait for the button to exist
       await allowWhileUsingButton.waitForExist({ timeout: 5000 });

       // Check if the button is displayed
       const isDisplayed = await allowWhileUsingButton.isDisplayed();

       if (isDisplayed) {
           console.log('Clicking "Allow While Using App"');
           await allowWhileUsingButton.click();
           return; // Exit the function after clicking
       }

       attempts++;

**Comment by christian-bromann** [comment]
> I am not able to interact with the google maps pop up,

Do you have a screenshot on how this looks like?
